pandas/tests/groupby/test_groupby.py

```diff
@@ -2521,6 +2521,34 @@ def test_groupby_cumprod(self):
         expected.name = 'value'
         tm.assert_series_equal(actual, expected)
 
+    def test_groupby_cum_skipna(self):
+        # GH 19806
```

Comment:
this needs to go with the other cumsum tests in pandas/tests/groupby/test_transform.py. put near the other cum* tests.

Reply:
Ok, I moved it to test_transform.py